### ISSUE TITLE
[FEATURE] Фичи, которых не хватает в Скальперском стакане

### ISSUE THREAD

**Issue by offvine** [alor-broker/Astras-Trading-UI#1547]
### Опишите проблему, которую решает ваша фича

1. Общий объем и дельта в панели кластеров прошедшие за выбранный временной интервал (ТФ)
2. бОльшая глубина стакана - минимум 50 (чем больше, тем лучше)

### Опишите предлагаемое вами решение

Предлагаю добавить

### Правила

- [X] Я согласен следовать правилам

**Comment by PilgrimViis** [comment]
@uncrashable по поводу №1 что думаешь?
По поводу №2 я скажу что в компании работают над этим. Но к сожалению это не в руках разработчиков Астраса. Последнее время на этот вопрос обратило внимание руководство и я думаю что в ближайшие полгода, сделают. Но это только моё мнение, не обещание.

**Comment by offvine** [comment]
По первому пункту - это стандартная штука которая есть, наверное, во всех "стаканных"/скальперских терминалах. Оценка проходящих объемов постоянно используется для принятия торговых решений.

**Comment by uncrashable** [comment]
> @uncrashable по поводу №1 что думаешь? По поводу №2 я скажу что в компании работают над этим. Но к сожалению это не в руках разработчиков Астраса. Последнее время на этот вопрос обратило внимание руководство и я думаю что в ближайшие полгода, сделают. Но это только моё мнение, не обещание.

Да, однозначно нужно, по крайней мере, суммарный объем. Подумаю как лучше сделать, отпишусь.
